src/main/scala/org/broadinstitute/hail/utils/richUtils/RichHadoopConfiguration.scala

```diff
@@ -107,7 +110,7 @@ class RichHadoopConfiguration(val hConf: hadoop.conf.Configuration) extends AnyV
       false, hConf)
   }
 
-  def copyMerge(srcFilenames: Array[String], destFilename: String, deleteSource: Boolean = true) {
+  def copyMerge(srcFilenames: Array[(String, (FileStatus) => Int)], destFilename: String, deleteSource: Boolean = true) {
```

Comment:
Ah, I see why you did it this way -- the header.  Could also take a `headerFile: Option[String] = None` argument and put that first.